projects/packages/connection/src/class-manager.php

```diff
@@ -451,6 +451,11 @@ private function internal_verify_xml_rpc_signature() {
 			return false;
 		}
 
+		// OAuth2 app authorization flow - skip signature verification.
```

Comment:
Should we add a comment that explains why we are doing this?
```suggestion
		// Skip XML-RPC signature verification for OAuth authorization flow.
		// OAuth uses GET requests without body-hash and has its own
		// signature verification in Authorize_Json_Api class.
```